packages/dds/tree/src/test/feature-libraries/objectForest.spec.ts

```diff
@@ -148,7 +148,7 @@ describe("object-forest", () => {
 		);
 	});
 
-	it("additional asserts validates schema after edit", () => {
+	it.skip("additional asserts validates schema after edit", () => {
```

Comment:
We should not disable document corruption detection nor tests that document corruption detection is working.

I'm pretty sure this PR corrupts documents, and we want to detect and fix that, not ignore it.